WORKSPACE

```diff
@@ -386,17 +373,3 @@ load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
 rbe_autoconfig(
     name = "buildkite_config",
 )
-
-# gazelle:repo bazel_gazelle
```

Comment:
This needs to remain in WORKSPACE. Otherwise running `bazel run //:gazelle` will fail to update BUILD files.

Reply:
fixed